.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: bash
+services: docker
+
+script:
+  - docker-compose up --build -d
```

Comment:
Using CI this way is a little bit misleading because the unit tests (though there are only a few for now) are not even run.
Travis has excellent Python support and resources to run Django natively.
Testing the docker build process should be an additional test at most.